### DISCUSSION TITLE
Set [build] passed/failed background color

### DISCUSSION THREAD

**Discussion by Mqxx** [badges/shields#9195] [Q&A] [answered]
Hey is it possible to set the background color of a build badge based on if the build has passed (green) or if it has failed (orange/red)?

**Comment by chris48s** [comment]
No. That is not an available customisation.

**Reply by Mqxx** [reply]
Okay thanks for the info!

**Reply by calebcartwright** [reply, marked as answer]
i'll also add that our default badges apply such coloring by default (red for failed, bright green for passing, etc.), but as Chris correctly notes we don't allow end users to control this. if you want something _different_ than our default coloring logic, then your best bet would be to utilize our [Custom Endpoint badge](https://shields.io/endpoint). Your endpoint could still use Shields for the status fetching (request the same badge url, but with a `.json` extension), and then apply whatever logic you want internally to set the color in your endpoint response

**Reply by Mqxx** [reply]
Okay thanks, I will look into this!
